scripts/generateMaintenanceTestData.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+header="R\"=("
```

Comment:
maybe a couple of lines about why this script exists

Reply:
Added FIXMEMAINTENANCE